Mage.Sets/src/mage/cards/w/WatcherOfTheSpheres.java

```diff
@@ -0,0 +1,65 @@
+package mage.cards.w;
+
+import mage.MageInt;
+import mage.abilities.common.EntersBattlefieldControlledTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.common.continuous.BoostSourceEffect;
+import mage.abilities.effects.common.cost.SpellsCostReductionControllerEffect;
+import mage.abilities.keyword.FlyingAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.SubType;
+import mage.constants.Zone;
+import mage.filter.FilterCard;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterControlledCreaturePermanent;
+import mage.filter.predicate.mageobject.AbilityPredicate;
+import mage.filter.predicate.permanent.AnotherPredicate;
+
+import java.util.UUID;
+
+/**
+ * @author jmharmon
+ */
+
+public final class WatcherOfTheSpheres extends CardImpl {
+
+    private static final FilterCard filter = new FilterCard("Creature spells with flying");
```

Comment:
you can have this be `FilterCreatureCard` so you don't have to add the creature predicate in the code block below

Reply:
Changed